docs/components/tuner/evaluation.md

```diff
@@ -167,6 +166,35 @@ print(metrics)
 }
 ```
 
+To customize the computed metrics, an optional `metrics` argument can be provided in
+the Evaluator constructor, that maps metric names to metric functions and keyword
```

Comment:
```suggestion
the Evaluator constructor, that maps metric names to metric functions and its keyword
```

Reply:
What do the keywords refer to here?

Reply:
So you specify a metric function that takes 2 arguments: binary relevance and max relevant items. All other arguments of the function should be keyword arguments and can be specified as well. For example all metric callables accept a keyword argument `k`